src/main/io/gps.c

```diff
@@ -780,39 +778,40 @@ static void ubloxSendNAV5Message(uint8_t model) {
     }
 }
 
-static void ubloxSendNavX5Message(void) {
-    ubxMessage_t tx_buffer;
-
-    if (gpsData.ubloxM9orAbove) {
-        uint8_t payload[1];
-        payload[0] = (uint8_t)(1 >> (8 * 0));
-        size_t offset = ubloxValSet(&tx_buffer, CFG_ANA_USE_ANA, payload, UBX_VAL_LAYER_RAM); // 5
-
-        ubloxSendConfigMessage(&tx_buffer, MSG_CFG_VALSET, offsetof(ubxCfgValSet_t, cfgData) + offset, true);
-    } else {
-        memset(&tx_buffer, 0, sizeof(ubxMessage_t));
-
-        tx_buffer.payload.cfg_nav5x.version = 0x0002;
-
-        tx_buffer.payload.cfg_nav5x.mask1 = 0x4000;
-        tx_buffer.payload.cfg_nav5x.mask2 = 0x0;
-        tx_buffer.payload.cfg_nav5x.minSVs = 0;
-        tx_buffer.payload.cfg_nav5x.maxSVs = 0;
-        tx_buffer.payload.cfg_nav5x.minCNO = 0;
-        tx_buffer.payload.cfg_nav5x.reserved1 = 0;
-        tx_buffer.payload.cfg_nav5x.iniFix3D = 0;
-        tx_buffer.payload.cfg_nav5x.ackAiding = 0;
-        tx_buffer.payload.cfg_nav5x.wknRollover = 0;
-        tx_buffer.payload.cfg_nav5x.sigAttenCompMode = 0;
-        tx_buffer.payload.cfg_nav5x.usePPP = 0;
-
-        tx_buffer.payload.cfg_nav5x.aopCfg = 0x1; //bit 0 = useAOP
-
-        tx_buffer.payload.cfg_nav5x.useAdr = 0;
-
-        ubloxSendConfigMessage(&tx_buffer, MSG_CFG_NAVX_SETTINGS, sizeof(ubxCfgNav5x_t), false);
-    }
-}
+// *** temporarily disabled for testing ***
```

Comment:
Are we certain at this point this is an actual pull request that people can review?

Reply:
Agree - still lots of testing going on as there are too many factors in play in this PR.

- All review comments needs to be resolved.
- We need to split up code in smaller files.
- Would help if we would make smaller changes in separated PR's.

Reply:
A major refactor is not the intent of this PR.  
Comment resolution will be achieved as the code fixes to address the comments are implemented.
General comments that do not provide code improvement suggestions will be marked resolved.